client/components/clonePost/index.jsx

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable no-underscore-dangle */
+/* global React, crNonce */
+
+const {
+  data: {
+    select,
+  },
+  i18n: {
+    __,
+  },
+} = wp;
+
+/**
+ * Slotfill to add clone & replace controls to the sidebar.
+ */
+const ClonePost = () => {
+  const currentPost = select('core/editor').getCurrentPost();
```

Comment:
This should be converted to a `useSelect` hook: https://developer.wordpress.org/block-editor/packages/packages-data/#useSelect

By using the hook, we don't need to use the `withSelect` HOC and we don't need to rely on extracting it from the `wp` object.